commands.go

```diff
@@ -34,6 +34,8 @@ func CmdAgent(ctx *cli.Context) {
 func runAgent() {
 	hostname, _ := os.Hostname()
 	ipaddress := getLocalIP()
+	filename := util.RandomString(10)
```

Comment:
hmm yeah, I think we should generate this filename & fullpath within bootstrap agent. is that possible?

Reply:
Done